src/definitions.jl

```diff
@@ -698,7 +699,7 @@ function adjoint_mul(p::Plan{T}, x::AbstractArray, ::RFFTAdjointStyle) where {T<
     pinv = inv(p)
     n = size(pinv, halfdim)
     # Optimization: when pinv is a ScaledPlan, fuse the scaling into our map to ensure we do not loop over x twice.
-    scale = pinv isa ScaledPlan ? pinv.scale / 2N : 1 / 2N
+    scale = pinv isa ScaledPlan ? pinv.scale / 2N : one(pinv.scale) / 2N
```

Comment:
This is wrong, the second branch will fail (and is apparently not tested): If `pinv` is not a `ScaledPlan`, `pinv.scale` does not exist in general.
```suggestion
    scale = pinv isa ScaledPlan ? pinv.scale / 2N : inv(2N)
```
Can you add a test that hits the second branch?

Reply:
Woah, thank you for the catch.